depthai_cmake/depthai_cmake_macros.cmake.in

```diff
@@ -102,7 +105,7 @@ macro(add_ros_node name)
 
     catkin_package (
       INCLUDE_DIRS ${_ARG_INCLUDE_DIRS}
-        LIBRARIES ${name}_nodelet
+        LIBRARIES ${name}_lib
```

Comment:
@youliangtan over here, the macro generates:
* new: `lib${name}_lib`  (unified library)
* old: `lib${name}_nodelet` and `lib${name}_lib`

2 thought on this:
* is there a compile time benefit associated with this as well?
* would `lib${name}` be a good choice for the library?

Reply:
Previously the `add_library()` is called twice for both `lib${name}_nodelet` and `lib${name}_lib`, which are both the same thing. Thus remove one of it.

I can definetly change the naming to `lib${name}`, but it will make some lines harder to read in `depthai_cmake_macros.cmake.in`. Anyway iam okay with both

Reply:
In `lib${name}` vs `lib${name}_lib`, I'd choose the former.  <-- minor bikeshed change

Between a nodelet + library and just nodelet and a separate library, it doesn't matter. We can keep the unified single library appraoch. Seems good to me :)

Reply:
Fixed https://github.com/osrf/depthai_ros/pull/4/commits/9110d8cf7d8f9d395c0fae2c501be41e29d96cd3